RIAssigner/compute/ComputationMethod.py

```diff
@@ -18,3 +18,6 @@ def _check_data_args(self, query, reference):
         """
         assert query is not None, "Query data is 'None'."
         assert reference is not None, "Reference data is 'None'."
+
+    def __eq__(self, o: object) -> bool:
+        return isinstance(o, type(self))

```

Comment:
Just a note to consider - if in the future you will possibly use nested inheritance 
(e.g. create a new Computation Method `XY` from `Kovats`, where you, for example, change only one of its methods). 

```python
class XY(Kovats):
    ...
```

Then, this kind of equality check `XY() == Kovats()` returns `True`, because `isinstance` considers also inheritance. Is that desired behavior? (See [this SO post](https://stackoverflow.com/a/1549814) for more detials)

A way to change this is to explicitelly compare their types, i.e.

```python
def __eq__(self, o: object) -> bool:
        return type(o) == type(self)
```

Reply:
Thanks for the comment! My linter complains about the type comparison with `type(o) == type(self)` and I am aware of the subclassing issue.

I don't think there will be computation methods derived from any of the derived classes, but this is good to know. I might violate the linter and change the type check to this one which seems to be stricter.

Reply:
What is the linter's output?